egs2/librispeech/wavlm1/local/path.sh

```
@@ -0,0 +1,24 @@
+torchaudio_plus(){
+    python3 <<EOF
+from packaging.version import parse as L
+if L('$torchaudio_version') >= L('$1'):
+    print("true")
+else:
+    print("false")
+EOF
+}
+
+if ! $(torchaudio_plus 0.13.1); then
+    echo "We recommend to run hubert with torchaudio version >= 0.13.1." >$2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)